documentation/src/main/markdown/downloadingandrunning/statelessscan.md

```diff
@@ -1,48 +1,48 @@
-# Ephemeral Scan LCA
+# Stateless Scan LCA
 
-Ephemeral Scan, or Ephemeral Scan Mode, is a new way of running [solution_name] with [blackduck_product_name]. This mode is designed to be as fast as possible and does not persist any data on [blackduck_product_name]. Ephemeral Scan Mode has a unique set of restrictions, mode of configuration and set of results.  It is similar to Rapid Scan Mode however it differs in that it supports usage of the SIGNATURE_SCAN tool.  Ephemeral Scan allows non-persistent signature scans to be performed.
+Stateless Scan, or Stateless Scan Mode, is a new way of running [solution_name] with [blackduck_product_name]. This mode is designed to be as fast as possible and does not persist any data on [blackduck_product_name]. Stateless Scan Mode has a unique set of restrictions, mode of configuration and set of results.  It is similar to Rapid Scan Mode however it differs in that it supports usage of the SIGNATURE_SCAN tool.  Stateless Scan allows non-persistent signature scans to be performed.
 
-Enable this feature by adding [--detect.blackduck.scan.mode=EPHEMERAL](../properties/configuration/blackduck-server.md#detect-scan-mode-advanced) to a run of Detect.
+Enable this feature by adding [--detect.blackduck.scan.mode=STATELESS](../properties/configuration/blackduck-server.md#detect-scan-mode-advanced) to a run of Detect.
 
 ## Requirements and Limitations
 
- * Ephemeral scanning is available under [blackduck_product_name] “Limited Customer Availability (LCA)”.
+ * Stateless scanning is available under [blackduck_product_name] “Limited Customer Availability (LCA)”.
  * Must be running [blackduck_product_name] 2022.10.0 or greater using the hosted KB.
  * Have Match as a Service (MaaS) enabled within [blackduck_product_name], a feature which will be available with the [blackduck_product_name] 2022.10.0 release.
  * A limited subset of Tools can be run.
     * The currently supported tools are: DETECTOR, BAZEL, SIGNATURE_SCAN and DOCKER.
-    * The EPHEMERAL Signature Scan will not persist on Black Duck. 
-    * All other tools are disabled when running in Ephemeral Scan mode.
- * Ephemeral Scan and non-persistent SIGNATURE_SCAN
-    * To perform a non-persistent Signature Scan in Ephemeral mode, SIGNATURE_SCAN must be included within --detect.tools.
+    * The STATELESS Signature Scan will not persist on Black Duck. 
```

Comment:
Maybe change this from "STATELESS" to "Stateless".

Reply:
I used case sensitive searches but I can sure change that to Stateless....